src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -55,7 +55,9 @@
    (log/info "Received Wallet Connect session proposal: " {:id (:id proposal)})
    (let [accounts                     (get-in db [:wallet :accounts])
          current-viewing-address      (get-in db [:wallet :current-viewing-account-address])
-         available-accounts           (wallet-connect-core/filter-operable-accounts (vals accounts))
+         available-accounts           (filter #(and (:operable? %)
+                                                    (not (:watch-only? %)))
+                                              (vals accounts))
```

Comment:
Why not to use `wallet-connect-core/filter-operable-accounts`?

Reply:
yeah it's weird. They look identical

Reply:
I'll fix it. I don't think I wrote this code. It's probably some remanent of a rebase.